base/server/src/main/java/org/dogtagpki/server/cli/SubsystemIdGeneratorUpdateCLI.java

```diff
@@ -310,4 +324,71 @@ private void updateRanges(DatabaseConfig dbConfig, LdapBoundConnection conn, Str
             conn.modify(serialDN, serialmod);
```

Comment:
Here we're modifying the `nextRange` used by SSNv1 values for SSNv2 so the admin will need to restore it to the original value if there's a migration issue. To avoid that I'd suggest moving it to SSNv2 subtree as well (i.e. `ou=ranges_v2`). We can address this in a separate PR.

Reply:
Yes, this will be in separate PR!